packages/core/src/app/order/OrderSummarySubtotals.test.tsx

```diff
@@ -37,31 +48,31 @@ describe('OrderSummarySubtotals', () => {
         );
 
         expect(screen.getByTestId('cart-taxes')).toHaveTextContent('Tax');
-        expect(screen.getByTestId('cart-taxes')).toHaveTextContent('$3.00');
+        expect(screen.getByTestId('cart-taxes')).toHaveTextContent('3');
 
         expect(screen.getByTestId('cart-subtotal')).toHaveTextContent('Subtotal');
-        expect(screen.getByTestId('cart-subtotal')).toHaveTextContent('$190.00');
+        expect(screen.getByTestId('cart-subtotal')).toHaveTextContent('190');
 
         expect(screen.getByTestId('cart-shipping')).toHaveTextContent('Shipping');
-        expect(screen.getByTestId('cart-shipping')).toHaveTextContent('$15.00');
+        expect(screen.getByTestId('cart-shipping')).toHaveTextContent('15');
 
         expect(screen.getByTestId('cart-handling')).toHaveTextContent('Handling');
-        expect(screen.getByTestId('cart-handling')).toHaveTextContent('$8.00');
+        expect(screen.getByTestId('cart-handling')).toHaveTextContent('8');
 
         expect(screen.getByTestId('cart-discount')).toHaveTextContent('Discount');
-        expect(screen.getByTestId('cart-discount')).toHaveTextContent('-$10.00');
+        expect(screen.getByTestId('cart-discount')).toHaveTextContent('-10');
 
         expect(screen.getByTestId('cart-gift-certificate')).toHaveTextContent('Gift Certificate');
-        expect(screen.getByTestId('cart-gift-certificate')).toHaveTextContent('-$7.00');
+        expect(screen.getByTestId('cart-gift-certificate')).toHaveTextContent('-7');
 
         const couponElements = screen.getAllByTestId('cart-coupon');
         const [firstCouponElement, secondCouponElement] = couponElements;
 
         expect(couponElements).toHaveLength(2);
-        expect(firstCouponElement).toHaveTextContent('-$5.00');
+        expect(firstCouponElement).toHaveTextContent('5');
         expect(firstCouponElement).toHaveTextContent('20% off each item');
         expect(firstCouponElement).toHaveTextContent('savebig2015');
-        expect(secondCouponElement).toHaveTextContent('-$5.00');
+        expect(secondCouponElement).toHaveTextContent('5');
```

Comment:
Those format should be out of scope and untouched.
Could we update `jest.mock('./OrderSummaryPrice')` to avoid uncessary changes?

Reply:
Hi Peng.
I am agree with you, this formatting and component in general should be out of scope for changes of child component.
In case of unit test we should test each unit separately. So if this format changed inside `OrderSummaryPrice` this logic should be tested inside unit test for `OrderSummaryPrice`.
And `OrderSummarySubtotals.test.tsx` should not test logic of child components or have any other dependency to child components. So this formatting check should not be here at all
But if you think that our unit test should be integration test, I will keep this formatting

Reply:
@bc-peng 
I rolled back this formatting.
Can you look, please?